discovery.md

```diff
@@ -523,9 +523,10 @@ as if that Service does not exist. For example, it would be excluded from
 any array of Services returned and it would result in a `404 Not Found`
 error for a request to that Service directly.
 
-Unless there is some out-of-band agreement, all APIs MUST use JSON encoding,
-which means when there is an HTTP Body the HTTP `Content-Type` Header MUST
-be `application/json`.
+All APIs MUST support JSON encoding, which means that an HTTP requests
+including an HTTP `Content-Type` Header of `application/json` and body in that
+format MUST be supported.  Likewise requests with `Accept` header of
+`application\json` MUST be supported.
```

Comment:
Of course everyone can make agreements to violate the specification and so long as they keep the arrangement that will be fine.